### PR TITLE
added updated config file for zenlink tests

### PR DESCRIPTION
**Pull request by neutrinoks**

### CODE REVIEW

**Review by sfffaaa — COMMENTED** [review]
Could you also add your modification in the config.parchain.test.yml? In my thought, people who want to run the test should use config.parachain.test.yml to launch their parachain in the future.

If this setting will use lots of resources, we can discuss it.

https://github.com/peaqnetwork/parachain-launch/blob/dev/config.parachain.test.yml

File: config.parachain.test.yml [hunk 1]
```diff
@@ -30,7 +39,8 @@ parachains:
     base: dev-local # the chain to use
     chainType: Development
     collators: # override collators
-      - ferdie
+      - alice # this imply //Alice
```
Comment (sfffaaa) [inline config.parachain.test.yml]: Could I ask why we want to use two collators, alice and bob here?

Comment (neutrinoks) [reply config.parachain.test.yml]: Shouldn't there be always two?! :thinking: but yeah, you are right, I don't want to be a ferdie-racist :blush:

**Review by sfffaaa — APPROVED** [review]

File: config.parachain.test.yml [hunk 2]
```diff
@@ -55,13 +63,44 @@ parachains:
...
-      - --name=relaychain-ferdie
-    env:
-      RUST_LOG: aura=debug # environment varitables for this node
+    - name: alice
+      wsPort: 9947
+      rpcPort: 9936
+      port: 31333
+      flags: # additional CLI flags for this node
+        # - --force-authoring
+        - --alice
```
Comment (sfffaaa) [inline config.parachain.test.yml]: Oops, this also needs to change to ferdie

**Review by sfffaaa — APPROVED** [review]

**Pull request merged**